ecs-logs-collector.sh

```diff
@@ -167,7 +167,7 @@ try_set_instance_infodir() {
   try "resolve instance-id"
 
   if command -v curl > /dev/null; then
-    instance_id=$(curl -s -m 3 http://169.254.169.254/latest/meta-data/instance-id 2>/dev/null)
+    instance_id=$(curl --max-time 900 -s -m 3 http://169.254.169.254/latest/meta-data/instance-id 2>/dev/null)
```

Comment:
The `-m` here (shortened version of `--max-time`, at least on my machine) should add a timeout, albeit much shorter than `900` seconds at a sensitive `3` seconds. I would prefer we use the `--long-option` form so let's drop the `-m` in favor of your `--max-time` :+1:

Are you seeing cases where the command got close to `900` seconds elapsed? If not, what do you think about `60` instead so as to not keep users on the edge of their seats? (Same for below change curl)

Reply:
sounds good, I will get rid of `-m` and change the timeout to 3.

Reply:
@jahkeup fixed